### PR TITLE
HOCS-3884 - Add document upload to appeals (Fix)

### PR DESCRIPTION
**Pull request by agraham19**
* Associate a single document with an appeal at completion

### CODE REVIEW

File: server/services/actions/types.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ module.exports = {
     ADD_APPROVAL_REQUEST: 'ADD_APPROVAL_REQUEST',
     EDIT_APPROVAL_REQUEST: 'EDIT_APPROVAL_REQUEST',
     ADD_CASE_APPEAL: 'ADD_CASE_APPEAL',
-    MANAGE_APPEAL_DOCUMENTS: 'MANAGE_APPEAL_DOCUMENTS',
+    ADD_APPEAL_DOCUMENT: 'MANAGE_APPEAL_DOCUMENTS',
```
Comment (gary-newbolt) [inline server/services/actions/types.js:29]: Did you want to change the value here for consistency as well?

File: server/services/forms/schemas/index.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = {
     addTemplate: require('./add-template'),
     managePeople: require('./manage-people'),
     contributionRequest: require('./contribution-request'),
-    manageAppealDocuments: require('./manage-appeal-documents'),
+    manageAppealDocuments: require('./add-appeal-document'),
```
Comment (gary-newbolt) [inline server/services/forms/schemas/index.js:24]: Again... does the key here need to be updated?

Comment (agraham19) [reply server/services/forms/schemas/index.js]: I have addressed this

File: server/services/actions/types.js (hunk 1 above)
Comment (agraham19) [reply server/services/actions/types.js:29]: I have addressed this

**Review by gary-newbolt — APPROVED** [review]
lgtm

**Pull request merged**